.github/workflows/tf-deploy-basic.yaml

```diff
@@ -1,42 +1,64 @@
 on:
   workflow_call:
     inputs:
-      account_id:
+      artifact_path:
```

Comment:
```suggestion
      github_artifact_path:
```
?

Reply:
(if renamed, then reference should be renamed too, I didn't  made suggestion when it's used)